JiraReporter/IndividualReportInfoService.cs

```diff
@@ -0,0 +1,69 @@
+﻿using JiraReporter.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace JiraReporter
+{
+    class IndividualReportInfoService
+    {
+        public void SetIndividualDraftInfo(List<Author> authors, SourceControlLogReporter.Model.Policy policy)
+        {
+            if(!policy.AdvancedOptions.NoIndividualDraft && policy.GeneratedProperties.IsIndividualDraft)
```

Comment:
Modifica in: 
`if (!ConditieActuala)  return;`

In asa fel incat cazul de baza sa fie in corpul metodei nu in cadrul unui if.